### PR TITLE
Updated CHANGELOG, platform version and client version.

### PR DESCRIPTION
**Pull request by simlay**
This finalizes the 0.7 release.

* `CHANGELOG` [rendered](https://github.com/simlay/fluvio/blob/0.5-stable/CHANGELOG.md)

### CODE REVIEW

**Review by sehz — APPROVED** [review]
Nice job

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.7.0-beta.1
```
Comment (simlay) [inline VERSION:1]: ugh, whyy new line you break my build

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
 
-## Client 0.40
+## Platform Version 0.7.0 - 2020-2-24
+* `fluvio cluster upgrade` ([#709](https://github.com/infinyon/fluvio/pull/709))
+* `install.sh` script works with `VERSION=latest` for prereleases([#812](https://github.com/infinyon/fluvio/pull/812))
+* Fix stream fetch ([#769](https://github.com/infinyon/fluvio/pull/769))
+* Limit for batchsize ([#787](https://github.com/infinyon/fluvio/pull/787))
+* Protocol updates ([#752](https://github.com/infinyon/fluvio/pull/752))
+* Socket close events propigate to client ([infinyon/fluvio-socket#22](https://github.com/infinyon/fluvio-socket/pull/22))
+* Fix sha265 has in `fluvio version` ([#740](https://github.com/infinyon/fluvio/pull/740))
```
Comment (nacardin) [inline CHANGELOG.md:10]: sha256?

**Review by nacardin — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (simlay) [reply CHANGELOG.md:10]: `sha256sum` or `shasum -a256` depending on your system.

Comment (simlay) [reply CHANGELOG.md:10]: oh, my typo, duh.

**Review by nacardin — APPROVED** [review]

**Pull request merged**